test_repro.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
was this intended to be checked in? it is at the root of the repo so we probably don't want it there.

Reply:
whoops, no it was not intended to be checked in